contrib/scalar/scalar.c

```diff
@@ -1030,9 +1098,18 @@ static int cmd_diagnose(int argc, const char **argv)
 	get_disk_info(&buf);
 	fwrite(buf.buf, buf.len, 1, stdout);
 
+	dir_file_stats(&packs_buf, ".git/objects/pack");
+	loose_objs_stats(&objs_buf, ".git/objects");
+
 	if ((res = stage(tmp_dir.buf, &buf, "diagnostics.log")))
 		goto diagnose_cleanup;
 
+	if ((res = stage(tmp_dir.buf, &packs_buf, "packs-local.txt")))
+		goto diagnose_cleanup;
+
+	if ((res = stage(tmp_dir.buf, &objs_buf, "objects-local.txt")))
+		goto diagnose_cleanup;
```

Comment:
Should these maybe just be....

```suggestion
	res = res | stage(tmp_dir.buf, &buf, "diagnostics.log");
	res = res | stage(tmp_dir.buf, &packs_buf, "packs-local.txt");
	res = res | stage(tmp_dir.buf, &objs_buf, "objects-local.txt");
```

...to do best-effort for all the items.

Reply:
Best effort on all items would probably be:

```
res = stage(...) | res;
```

😄